### PR TITLE
Docker Desktop 4.6.1 release notes

### PR DESCRIPTION
**Pull request by ebriney**
### Proposed changes

Added Docker Desktop 4.6.1 release notes.

### CODE REVIEW

File: desktop/mac/release-notes/index.md [hunk 1]
```diff
@@ -33,6 +33,24 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
 > .button .primary-btn }
 
+### Upgrades
+
+- [Buildx 0.8.1](https://github.com/docker/buildx/releases/tag/v0.8.1)
+
+### Bug fixes and minor changes
+
+- Fixed diagnostics upload when there is no HTTP proxy set.
+- Remove a false positive "vm is not running" error from self-diagnose.
```
Comment (mat007) [inline desktop/mac/release-notes/index.md:43]: Maybe
```suggestion
- Removed a false positive "vm is not running" error from self-diagnose.
```
?

Also there is https://github.com/docker/for-mac/issues/6233 to link to.

File: desktop/mac/release-notes/index.md [hunk 2]
```diff
@@ -33,6 +33,24 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Avoid spinning in vpnkit-forwarder
```
Comment (mat007) [inline desktop/mac/release-notes/index.md:44]:
```suggestion
- Prevented spinning in vpnkit-forwarder filling the logs with error messages.
```
?

File: desktop/mac/release-notes/index.md [hunk 3]
```diff
@@ -33,6 +33,24 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Upgrades
+
+- [Buildx 0.8.1](https://github.com/docker/buildx/releases/tag/v0.8.1)
+
+### Bug fixes and minor changes
+
+- Fixed diagnostics upload when there is no HTTP proxy set.
```
Comment (mat007) [inline desktop/mac/release-notes/index.md:42]: We actually have an issue to link there: https://github.com/docker/for-mac/issues/6234

File: desktop/windows/release-notes/index.md [hunk 4]
```diff
@@ -23,15 +23,32 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
-## Docker Desktop 4.6.0
-2022-03-14
+## Docker Desktop 4.6.1
+2022-03-21
```
Comment (mat007) [inline desktop/windows/release-notes/index.md:27]:
```suggestion
2022-03-22
```
now 😬

File: desktop/mac/release-notes/index.md [hunk 5]
```diff
@@ -33,6 +33,24 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Upgrades
+
+- [Buildx 0.8.1](https://github.com/docker/buildx/releases/tag/v0.8.1)
```
Comment (mat007) [inline desktop/mac/release-notes/index.md:38]: Should we be explaining what this fixes? I haven’t looked at the changes to be honest.

File: desktop/mac/release-notes/index.md [hunk 6]
```diff
@@ -23,8 +23,8 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
-## Docker Desktop 4.6.0
-2022-03-14
+## Docker Desktop 4.6.1
+2022-03-21
```
Comment (usha-mandya) [inline desktop/mac/release-notes/index.md:27]:
```suggestion
2022-03-22
```

File: desktop/mac/release-notes/index.md (hunk 5 above)
Comment (ebriney) [reply desktop/mac/release-notes/index.md:38]: I checked a bit over release notes and we just put the upstream tag usually

**Review by mat007 — APPROVED** [review]
Looks good to me!

**Review by usha-mandya — APPROVED** [review]

**Comment by mat007** [comment]
We’re ready!

**Pull request merged**